Formula/aws-nuke.rb

```diff
@@ -0,0 +1,45 @@
+class AwsNuke < Formula
+  desc "Nuke a whole AWS account and delete all its resources"
+  homepage "https://github.com/rebuy-de/aws-nuke"
+  url "https://github.com/rebuy-de/aws-nuke.git",
+      tag:      "v2.19.0",
+      revision: "357aa44f5a04df9d1faa8c57bb5ce924871a910a"
+  license "MIT"
+  head "https://github.com/rebuy-de/aws-nuke.git", branch: "main"
+
+  depends_on "go" => :build
+
+  resource("example_config") do
+    url "https://raw.githubusercontent.com/rebuy-de/aws-nuke/5da46161d4d222216c6928bc4c47406cd9bb4691/config/example.yaml"
+    sha256 "00b476307cf0b9f6ae6bde14af9f237afa9c2225cbb0125171863f43b74fd184"
+  end
+
+  def install
+    major_version = version.major
+    build_xdst="github.com/rebuy-de/aws-nuke/v#{major_version}/cmd"
+    ldflags = %W[
+      -s -w
+      -X #{build_xdst}.BuildVersion=#{version}
+      -X #{build_xdst}.BuildDate=#{time.strftime("%F")}
+      -X #{build_xdst}.BuildHash=#{Utils.git_head}
+      -X #{build_xdst}.BuildEnvironment=homebrew
+    ]
+    with_env(
+      "CGO_ENABLED" => "0",
+    ) do
+      system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/aws-nuke", "."
```

Comment:
Those are the default 
```suggestion
      system "go", "build", *std_go_args(ldflags: ldflags)
```